src/main/resources/templates/stores.html

```diff
@@ -27,35 +27,56 @@
             <li>
               <a href="#" class="header__menu-item">about</a>
             </li>
-            <li>
-              <a href="catalog" class="header__menu-item">catalog</a>
+            <li th:switch="${userRole}">
+              <div th:case="null">
+                <a href="catalog" class="header__menu-item">catalog</a>
+              </div>
+              <div th:case="*">
+                <a href="catalog" class="header__menu-item">wantlist</a>
+              </div>
             </li>
             <li>
-              <a href="index.html" class="header__logo">
+              <a href="/" class="header__logo">
                 <img src="img/svg/wax_deals.svg" alt="wax deals logo">
               </a>
             </li>
             <li>
-              <a href="#" class="header__menu-item header__menu-item_active">stores</a>
+              <a href="#" class="header__menu-item">stores</a>
```

Comment:
it should be leave here, because STORES must be orange on if user on this page

Reply:
Could you please explain, as I can see there is no the "header__menu-item_active" class in the develop branch